libres/lib/analysis/ies/ies_enkf_config.cpp

```diff
@@ -75,7 +77,6 @@ ies_enkf_config_type *ies_enkf_config_alloc() {
     ies_enkf_config_set_ies_subspace(config, DEFAULT_IES_SUBSPACE);
     ies_enkf_config_set_ies_inversion(config, DEFAULT_IES_INVERSION);
     ies_enkf_config_set_ies_logfile(config, DEFAULT_IES_LOGFILE);
-    ies_enkf_config_set_ies_debug(config, DEFAULT_IES_DEBUG);
```

Comment:
This would be a breaking change in the input config file,  so I don't think introducing breaking changes for the christmas release will be well received, if it impacts any of our users. Maybe we should introduce logging or warnings before making this change. Any input @oyvindeide?

Reply:
The entire `bool dbg;` is removed - it was a pure development aid.

Reply:
Agree that we should introduce a warning here, with a version for when it will be removed (in the near future). We have previously removed a few keywords, but the result of that has been that cases have printet out that it is not recognised. Adding a warning will make it consistent at least.

Reply:
OK - I have done as I have suggested; if the user tries to invoke the `IES_DEBUG` subsetting a warning will be issued.